geowebcache-webapp/README.md

```diff
@@ -0,0 +1,41 @@
+# GeoWebCache distribution for geOrchestra
+
+This is a geOrchestra specific re-packaging of GeoWebcache with customized extensions to
+
+- Handle authentication through geOrchestra's authentication HTTP request headers
+- Use a custom URL "mangler" for GWC to return URL's based on geOrchestra's public URL
+
+## Build
+
+```
+mvn clean install -f geowebcache-webapp
```

Comment:
`-f` option expects an alternate pom.xml file, did you mean `-f geowebcache-webapp/pom.xml` or `-pl geowebcache-webapp` ? relying on the makefile modification above, I'd expect the second one.

Reply:
Considering the pom modification below (moving the parent from org.georchestra.root to geowebcache), I think the -f would be more likely to work, as -pl is likely to work on children maven project ?